DataConnectors/Ubiquiti/Connector_Ubiquiti_agent.json

```diff
@@ -0,0 +1,159 @@
+            "description": "Download the agent on the relevant machine and follow the instructions.", 
+            "instructions": [ 
+                { 
+                "parameters": { 
+                "linkType": "InstallAgentOnLinuxNonAzure" 
+                }, 
+                "type": "InstallAgent" 
+                } 
+            ] 
+        } 
+    ] 
+        }, 
+            "type": "InstructionStepsGroup" 
+            } 
+        ] 
+            }, 
+            { 
+                "instructions": [ 
+                    { 
+                    "parameters": { 
+                    "title": "Choose where to install the Windows agent:", 
+        "instructionSteps": [ 
+                { 
+                "title": "Install agent on Azure Windows Virtual Machine", 
+                "description": "Select the machine to install the agent on and then click **Connect**.", 
+                "instructions": [ 
+                    { 
+                    "parameters": { 
+                    "linkType": "InstallAgentOnVirtualMachine"
+                    }, 
+                    "type": "InstallAgent" 
+                    }
+                ]    
+            }, 
+                { 
+                "title": "Install agent on a non-Azure Windows Machine", 
+                "description": "Download the agent on the relevant machine and follow the instructions.", 
+                "instructions": [ 
+                    { 
+                    "parameters": { 
+                    "linkType": "InstallAgentOnNonAzure"  
+                    }, 
+                    "type": "InstallAgent" 
+                    } 
+                ] 
+            } 
+        ] 
+            }, 
+                "type": "InstructionStepsGroup" 
+                } 
+            ] 
+                }, 
+    { 
+            "title": "2. Configure the logs to be collected", 
+            "description":"Follow the configuration steps below to get Ubiquiti logs into Azure Sentinel. Refer to the [Azure Monitor Documentation](https://docs.microsoft.com/azure/azure-monitor/agents/data-sources-json) for more details on these steps.\n1. Configure log forwarding on your Ubiquiti controller(Refer to [Ubiquiti Comunity](https://community.ui.com/) for detailed instructions). 2. Download config file [Ubiquiti.conf](https://aka.ms/sentinel-UbiquitiUnifi-conf).\n3. Login to the server where you have installed Azure Log Analytics agent.\n4. Copy Ubiquiti.conf to the /etc/opt/microsoft/omsagent/**workspace_id**/conf/omsagent.d/ folder.\n5. Edit Ubiquiti.conf as follows:\n\n\t i. specify port which you have set your Ubiquity device to forward logs to (line 4)\n\n\t ii. replace **workspace_id** with real value of your Workspace ID (lines 14,15,16,19)\n5. Save changes and restart the Azure Log Analytics agent for Linux service with the following command:\n\t\tsudo /opt/microsoft/omsagent/bin/service_control restart",

```

Comment:
@vu-socprime : Ubiquiti connector is referring to a link (https://aka.ms/sentinel-UbiquitiUnifi-conf) resolving into ubiquiti.conf file, can you please provide it?

Reply:
@socprime - Can you please give us ubiquiti.conf file? Need that to complete Ubiquiti solution. Thanks.